msteams-platform/apps-in-teams-meetings/build-apps-for-teams-meeting.md

```diff
@@ -0,0 +1,205 @@
+---
+title: Build apps for Teams meeting stage
+author: v-sdhakshina
+description: In this article, learn how to build meeting apps API references that are available for Teams client.
+ms.topic: conceptual
+ms.author: v-sdhakshina
+ms.localizationpriority: medium
+ms.date: 04/07/2022
+---
+
+# Build apps for Teams meeting stage
+
+Share to stage allows users to share an app to the meeting stage from the app side panel in an ongoing meeting. This sharing is interactive & collaborative as opposed to passive screen sharing.
+
+Users can invoke share to stage by selecting on the **Share to Stage** icon on the top right side of the app side panel. This is a native Teams client button and selecting it shares the entire app to the meeting stage.  
+
+## App manifest settings for apps in meeting stage
+
+To share an app to the meeting stage, the app must call the meeting side panel experience in manifest setting:
+
+```json
+"context":[ 
+    "meetingSidePanel", 
+    "meetingStage" 
+     ] 
+```
+
+## Advanced share to stage APIs
+
+There are many scenarios where sharing the entire app to the meeting stage isn't as useful as sharing specific parts of the app:  
+
+1. For a brainstorming or whiteboard app, a user may want to share a specific board in a meeting vs the entire app with all the boards.  
+
+1. For a medical app, a doctor may want to share just the X-Ray on the screen with the patient vs sharing the entire app with all the patients' records or results etc.  
+
+1. For an app like Arcade, the user would want to share individual games to the meeting stage vs the entire Arcade app.  
```

Comment:
> For an app like Arcade

Is this released already? Will readers know what we're referring to here?

Reply:
Good point @aosolis checking with PR what other app can we use. Thanks for catching it.

Reply:
@v-sdhakshina lets remove this line since arcade is not GAed yet.